.github/test_e2e_single_chain_op_succinct_aggoracle_committee_args.json

```diff
@@ -19,6 +19,24 @@
     ],
     "use_agg_oracle_committee": true,
     "agg_oracle_committee_quorum": 2,
-    "agg_oracle_committee_total_members": 3
+    "agg_oracle_committee_total_members": 3,
+    "zkevm_rollup_chain_id": 20201
+  },
+  "optimism_package": {
+    "predeployed_contracts": true,
```

Comment:
same here, we use pre-deployed contracts by default: https://github.com/0xPolygon/kurtosis-cdk/blob/2c749117bb7078c361cf31542968224e00791ea3/src/package_io/op_input_parser.star#L96

Reply:
Fixed https://github.com/agglayer/aggkit/pull/1128/commits/f4e3fce4d0cdb8df5009a658592198c191f26596